src/main/resources/application.conf

```diff
@@ -4,7 +4,12 @@ ktor {
         port = ${?PORT}
     }
     application {
-        modules = [ avans.avd.ApplicationKt.module ]
+        modules = [
+            avans.avd.ApplicationKt.module,
+            avans.avd.auth.AuthModuleKt.authModule,
+            avans.avd.incidents.IncidentsModuleKt.incidentsModule,
+            avans.avd.users.UsersModuleKt.usersModule,
```

Comment:
I split up the application modules by feature, so now you can potentially break up your service into smaller microservices just by removing these modules.